src/components/pages/CompanySubscriptions/index.tsx

```diff
@@ -108,21 +125,36 @@ export default function CompanySubscriptions() {
     return validateExpr
   }
 
+  const callSuccess = () => {
+    success(t('content.organization.unsubscribe.unsubscribeSuccess'))
+    setLoading(false)
+    setShowUnsubscribeOverlay(false)
+    setEnableErrorMessage(false)
+    setRefresh(Date.now())
+  }
+
   const onUnsubscribeSubmit = async () => {
     setLoading(true)
-    await unsubscribeMutation(subscriptionId)
-      .unwrap()
-      .then(() => {
-        success(t('content.organization.unsubscribe.unsubscribeSuccess'))
-        setLoading(false)
-        setShowUnsubscribeOverlay(false)
-        setEnableErrorMessage(false)
-        setRefresh(Date.now())
-      })
-      .catch(() => {
-        setLoading(false)
-        setEnableErrorMessage(true)
-      })
+    if (currentActive === 0)
+      await unsubscribeAppMutation(subscriptionId)
+        .unwrap()
+        .then(() => {
+          callSuccess()
+        })
+        .catch(() => {
+          setLoading(false)
+          setEnableErrorMessage(true)
+        })
+    else
+      await unsubscribeServiceMutation(subscriptionId)
+        .unwrap()
+        .then(() => {
+          callSuccess()
+        })
+        .catch(() => {
+          setLoading(false)
+          setEnableErrorMessage(true)
+        })
```

Comment:
too much duplication because if and else block almost identical - simplify: 

```
await (currentActive === 0 ? unsubscribeAppMutation : unsubscribeServiceMutation)(subscriptionId)
  .unwrap()
  ...
```